Makefile

```diff
@@ -160,7 +160,7 @@ test-unit: install-deps
 test-integration: install-deps build-python-packages	
 	cd $(TESTS_DIR)
 	$(POETRY) run $(PYTHON) -m env_control create
-	$(POETRY) run behave --show-timings -D skip_setup --junit $(BEHAVE_ARGS)
+	$(POETRY) run behave --show-timings --stop -D skip_setup --junit $(BEHAVE_ARGS)
```

Comment:
Added `--stop` for consistency with behave options in https://github.com/yandex/ch-backup. Also it simplifies debugging a lot.